### PR TITLE
Reusable workflow testing

### PR DESCRIPTION
**Pull request by daler**
This PR is for working out the details of how reusable workflows for container building might work to address #64 and to make building {arm,amd}64 containers more straightforward -- and therefore more sustainable for community maintenance.

### CODE REVIEW

File: .github/workflows/generic_build.yml [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+          rm "${iidfile}"
+
+          # And then extend the build args with this image.
+          BUILD_ARGS+=("--build-arg=busybox_image=${busybox_image}")
+        fi
+
+        # ----------------------------------------------------------------------
+
+        # Build each arch's image
+        for arch in $archs_and_images ; do
```
Comment (martin-g) [inline .github/workflows/generic_build.yml]: Currently only an AMD64 image is added to the manifest - https://pipelinesghubeus24.actions.githubusercontent.com/0YHXh0QGzpCiRgaqrcqmDx4aoYjcAOBDwhzeyyC598TlkSuHqS/_apis/pipelines/1/runs/345/signedlogcontent/2?urlExpires=2024-02-05T15%3A40%3A20.3157368Z&urlSigningMethod=HMACV1&urlSignature=hK6kHUewgqlYl8%2BGIobY20GzaihVEyRhvkhn2ZBPr00%3D.
I think this should be `for arch in "${archs_and_images[@]}"; do` to iterate over the strings in the Bash array.

**Comment by martin-g** [comment]
Do you need any help with this ?
Everything looks good to me!
What's the next step ?

**Comment by daler** [comment]
@martin-g I just commented over in https://github.com/bioconda/bioconda-containers/issues/74#issuecomment-1934528159 to keep the info contained within that issue.

**Comment by daler** [comment]
Closing in favor of https://github.com/bioconda/bioconda-utils/pull/953; see #74 for rationale.

**Pull request closed without merging**